examples/skills_tuning/instructlab/unstructured_to_structured.py

```
@@ -0,0 +1,284 @@
+
+# Third Party
+from datasets import Dataset
+from openai import OpenAI
+from rich import print
+from rich.console import Console
+from rich.panel import Panel
+from rich.syntax import Syntax
+import yaml
+
+# First Party
+import sdg_hub
```

Flake8 (7.2.0)

[error] 'sdg_hub' imported but unused

(F401)